src/eduid_userdb/user.py

```diff
@@ -51,6 +52,8 @@
 
 VALID_SUBJECT_VALUES = ['physical person']
 
+U = TypeVar('U', bound='User')
```

Comment:
This would lead to error messages about something not being a 'U' - pretty Useless :)

There seems to be a convention to start TypeVar's with T - and since this one has an upper bound on User I would suggest the name TUserSubclass.

Reply:
Corrected. I actually thought that the convention was to use single letters, like in rust, when I saw a commit of yours yesterday with a TypeVar I thought of telling you! =)